packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts

```diff
@@ -187,12 +187,23 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
     }
 
     if (existingFieldMetadata.isCustom === false) {
+      let fieldMetadataInputOverrided = {};
+
+      if (existingFieldMetadata.type === FieldMetadataType.SELECT) {
+        fieldMetadataInputOverrided = {
+          options: fieldMetadataInput.options,
+        };
+      }
+
       // We can only update the isActive field for standard fields
-      fieldMetadataInput = {
+      fieldMetadataInputOverrided = {
+        ...fieldMetadataInputOverrided,
         id: fieldMetadataInput.id,
         isActive: fieldMetadataInput.isActive,
         workspaceId: fieldMetadataInput.workspaceId,
       };
+
+      fieldMetadataInput = fieldMetadataInputOverrided as UpdateFieldInput;
```

Comment:
@thomtrp As this part is becoming bigger, maybe we should create an utility function ?

Reply:
I exported it as a separated function and re-ordered the global function:
- check if the input is valid
- override input if the field is standard
- perform update